### PR TITLE
Use all the styles defined and display geometries coming from style

### PR DESCRIPTION
**Pull request by oterral**
All in the title.

### CODE REVIEW

**Comment by gberaudo** [comment]
Great @oterral, thanks for this contribution.

File: examples/vectors.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
       onclick="javascript:scene.globe.depthTestAgainstTerrain = !scene.globe.depthTestAgainstTerrain; ol3d.getAutoRenderLoop().restartRenderLoop()" /></div>
     <div><input type="button" value="Toggle circle style"
       onclick="javascript:toggleStyle()" /></div>
+    <div><input type="button" value="Replace 'Only text' by a circle using style with a custom geometry"
+      onclick="javascript:toggleGeomFuncStyle()" /></div>
```
Comment (gberaudo) [inline examples/vectors.html]: The function is not defined.

**Comment by oterral** [comment]
Unneeded stufs in vectors example removed

**Comment by gberaudo** [comment]
Thanks @oterral. Just realized after merging that the changelog was not updated. Can you create a new PR with it?

**Comment by oterral** [comment]
yes.

**Pull request merged**